client/state/sites/selectors/get-site-collisions.js

```diff
@@ -11,16 +11,5 @@ import getSitesItems from 'calypso/state/selectors/get-sites-items';
  */
 export default createSelector( ( state ) => {
 	const sitesItems = Object.values( getSitesItems( state ) );
-	return sitesItems
-		.filter( ( site ) => {
-			const siteUrlSansProtocol = withoutHttp( site.URL );
-			return (
-				! site.jetpack &&
-				sitesItems.some(
-					( jetpackSite ) =>
-						jetpackSite.jetpack && siteUrlSansProtocol === withoutHttp( jetpackSite.URL )
-				)
-			);
-		} )
-		.map( ( site ) => site.ID );
```

Comment:
Moving this function to `client/lib/site/utils.js` to be able to reuse it out of selectors.

Reply:
> Moving this function to `client/lib/site/utils.js` to be able to reuse it out of selectors.

Great idea 😄